Assets/VRTK/Scripts/Presence/VRTK_BodyPhysics.cs

```diff
@@ -869,22 +872,44 @@ protected virtual void CreateCollider()
                 bodyRigidbody.freezeRotation = true;
             }
 
-            if (bodyColliderContainer == null)
+            if (customBodyCollider == null)
             {
                 generateCollider = true;
-                bodyColliderContainer = CreateColliderContainer(BODY_COLLIDER_CONTAINER_NAME, playArea);
+                customBodyCollider = CreateColliderContainer(BODY_COLLIDER_CONTAINER_NAME, playArea);
 
-                bodyCollider = bodyColliderContainer.AddComponent<CapsuleCollider>();
+                bodyCollider = customBodyCollider.AddComponent<CapsuleCollider>();
                 bodyCollider.radius = bodyRadius;
 
+                customBodyCollider.gameObject.layer = LayerMask.NameToLayer("Ignore Raycast");
+            }
+            else
+            {
+                customBodyCollider.transform.SetParent(playArea);
+                bodyCollider = customBodyCollider.GetComponent<CapsuleCollider>();
+                VRTK_PlayerObject.SetPlayerObject(customBodyCollider, VRTK_PlayerObject.ObjectTypes.Collider);
+                if(!bodyCollider)
+                {
+                    bodyCollider = customBodyCollider.AddComponent<CapsuleCollider>();
+                    bodyCollider.radius = bodyRadius;
+                }
+            }
+            if (customFootCollider == null)
+            {
                 if (CalculateStepUpYOffset() > 0f)
                 {
-                    GameObject footColliderContainer = CreateColliderContainer(FOOT_COLLIDER_CONTAINER_NAME, bodyColliderContainer.transform);
-                    footCollider = footColliderContainer.AddComponent<CapsuleCollider>();
+                    customFootCollider = CreateColliderContainer(FOOT_COLLIDER_CONTAINER_NAME, customBodyCollider.transform);
+                    footCollider = customFootCollider.AddComponent<CapsuleCollider>();
+                }
+            }
+            else
+            {
+                customFootCollider.transform.SetParent(customBodyCollider.transform);
```

Comment:
This still creates the custom foot collider even if the step up Y offset is `0`, the above code does a check to see if it's above 0f